Remora.Commands/Remora.Commands.csproj

```diff
@@ -19,6 +19,8 @@
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
+      <PackageReference Include="OneOf"/>
       <PackageReference Include="Remora.Results"/>
+      <PackageReference Update="Remora.Results.Analyzers"/>
```

Comment:
What's up with this?

Reply:
I'm...Just as confused as you, really. I'll remove it; dunno where it sneaked in from.